src/core/test/system/system.t.sol

```diff
@@ -258,8 +230,8 @@ contract SystemTest is DSTest {
         return extra;
     }
 
-    function currLenderBal() public returns(uint) {
-        return tkn.balanceOf(lenderTokenAddr(address(deployer.lender())));
+    function currDeskBal() public returns(uint) {
```

Comment:
I think we need to refactor it later because the desk won't hold currency. 
It will be in different tranches but I think it fine for now.

Reply:
I will leave it like this for now but make a note to refactor when we have the rest of the lender side contracts in.